integration/http/pom.xml

```diff
@@ -55,6 +59,13 @@
             <artifactId>fastjson</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
I think should not import the spring-boot dependency in normal module. And move the HttpAutoConfiguration to spring-boot-starter module will be better.

Reply:
I have move the code to seata-spring-boot-starter model ,please check it